factory/README.md

```diff
@@ -0,0 +1,201 @@
+# cypress/factory
+
+`cypress/factory` is a docker image that can be used with docker [`args`](https://docs.docker.com/engine/reference/builder/#arg) to generate a docker container with specific versions of:
+
+* node
+* yarn
+* chrome
+* firefox
+* edge
+* cypress
+
+## Benefits
+
+* Freedom to choose which versions to test against.
+* No need to wait on an official release to test the latest version of a browser.
+* Smaller docker sizes especially when not including unused browsers.
+* Easily test multiple browser versions.
+* Reduced maintenance and pull requests for the cypress-docker repo.
+* Ability to offer more variations of docker containers at low cost.
```

Comment:
Just for my edification, what is the cost incurred here (which is potentially now reduced)?

Reply:
This is a potential cost savings. If we want to offer say a container that contains just cypress and node, previously we'd have to write a new script, make new folders and manually update each version. With the new CI and .env setup all we have to do is update CI and a couple of docker-compose files and new versions would be released automatically in sync with the `included` image based on the version of cypress and node in the .env file.

Reply:
That's what I assumed it implied. It's really an internal cost/time savings. Its minor, but may be worth clarifying (I think it speaks loudly to a fundamental impact here), to something like _Ability for Cypress to offer_ so its a known savings commodity that we'd be paying forward.

Reply:
```suggestion
* Ability for Cypress to offer more variations of docker containers at low cost.
```